TOT.Entities/Policy_Entities/AccrualSchedule.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace TOT.Entities.Policy_Entities
+{
+    public class AccrualSchedule
```

Comment:
Как я писал в https://github.com/ASbeletsky/TimeOffTracker/issues/8#issuecomment-431662507 я предлагаю:
```suggestion
    public class TimeOffPolicy
    {
         public int Id { get; set; }
         public string Name { get; set; }
         public TimeOffType Type { get; set; }
         public int TimeOffDaysPerYear { get; set;}
    }
```
Для начала пусть будет так, потом сделаем более гибко, если будет время